examples/README.md

```diff
@@ -10,16 +10,16 @@ your cluster. If you are following instructions [here](../DEVELOPMENT.md#getting
 
 ```bash
 # To invoke the build-push Task only
-kubectl apply -f examples/taskruns/taskrun.yaml
+kubectl apply -f taskruns/taskrun.yaml
```

Comment:
usually we assume that a user is running the `kubectl` commands from the root of the repo and that's why the README has the `examples` as part of the path

Reply:
I think the knative doc should follow a unified way to clarify how to use the command, I found that most of the document are assuming if the README and YAML files in same dir, then we can use the relative path, take a look at here for example https://github.com/knative/docs/tree/master/eventing/samples/cronjob-source#cron-job-source-example

Reply:
I second @pivotal-nader-ziada  thoughts on maintaining the pattern of `kubectl` commands with root of the repo.